Sources/StreamChatUI/UIConfig+MessageList.swift

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit.UIImage
+
+public extension _UIConfig {
+    struct MessageListUI {
+        public var messageListVC: _ChatMessageListVC<ExtraData>.Type = _ChatMessageListVC<ExtraData>.self
+        public var incomingMessageCell: _СhatMessageCollectionViewCell<ExtraData>.Type =
+            СhatIncomingMessageCollectionViewCell<ExtraData>.self
+        public var outgoingMessageCell: _СhatMessageCollectionViewCell<ExtraData>.Type =
+            СhatOutgoingMessageCollectionViewCell<ExtraData>.self
+
+        public var collectionView: ChatMessageListCollectionView.Type = ChatMessageListCollectionView.self
+        public var collectionLayout: ChatMessageListCollectionViewLayout.Type = ChatMessageListCollectionViewLayout.self
+        public var channelNamer: ChatChannelNamer.Type = ChatChannelNamer.self
+        public var minTimeInvteralBetweenMessagesInGroup: TimeInterval = 10
+        /// Vertical contentOffset for message list, when next message batch should be requested
+        public var offsetToPreloadMoreMessages: CGFloat = 100
```

Comment:
If we say that the rule is that only contestants that are shared between multiple views should be in `UIConfig`, I think these two probably shouldn't be here.

Reply:
Fixed. Thanks :)